doc/specs/stdlib_hashmaps.md

```diff
@@ -584,6 +588,75 @@ an allocatable of `class(*)`. It is an `intent(out)` argument.
     end program demo_get
 ```
 
+#### `get_other_scalar` - extracts a scalar value from a derived type
+
+##### Status
+
+Experimental
+
+##### Description
+
+Extracts a scalar value from a `other_type` and stores it in the scalar variable
+`value`.
+
+##### Syntax
+
+`call [[stdlib_hashmap_wrappers:get_other_scalar]]( other[, value_char,
+value_int8, value_int16, value_int32, value_int64, value_sp, value_dp, value_csp, value_cdp, value_lk,
+exists] )`
```

Comment:
I very much dislike this API. In which case do you need to retrieve both a `value_sp` and `value_lk` together? The user would have to implement the same dispatch logic again which is already used in the wrapper.

Reply:
Reading it again now, I totally agree with you. I will change the code with multiple subroutines.

Reply:
For TOML Fortran I'm using the following `get_value` API: https://github.com/toml-f/toml-f/blob/main/src/tomlf/build/table.f90